schemas/com.snowplowanalytics.badrows/schema_violations/jsonschema/1-0-0

```diff
@@ -0,0 +1,167 @@
+                  },
+                  "instance": {
+                    "type": "string",
+                    "description": "Where the offending data was found",
+                    "maxLength": 128
+                  },
+                  "igluRepositories": {
+                    "type": "array",
+                    "description": "Iglu repositories tried or where the offending schema was found",
+                    "items": {
+                      "type": "string",
+                      "description": "Name of the Iglu repository",
+                      "maxLength": 128
+                    }
+                  },
+                  "message": {
+                    "type": "string",
+                    "description": "Failure message associated with this schema violation",
+                    "maxLength": 256
+                  }
+                },
+                "required": [ "schemaKey", "instance", "igluRepositories", "message" ],
+                "additionalProperties": false
+              },
+              {
+                "properties": {
+                  "schemaKey": {
+                    "type": "string",
+                    "description": "Schema which was violated",
+                    "pattern": "^iglu:[a-zA-Z0-9-_.]+/[a-zA-Z0-9-_]+/[a-zA-Z0-9-_]+/[0-9]+-[0-9]+-[0-9]+$"
+                  },
+                  "instance": {
+                    "type": "string",
+                    "description": "Where the offending data was found",
+                    "maxLength": 128
+                  },
+                  "igluRepositories": {
+                    "type": "array",
+                    "description": "Iglu repositories tried or where the offending schema was found",
+                    "items": {
+                      "type": "string",
+                      "description": "Name of the Iglu repository",
+                      "maxLength": 128
+                    }
+                  },
+                  "property": {
+                    "type": "string",
+                    "description": "Property which caused the schema violation",
+                    "maxLength": 256
+                  },
+                  "value": {
+                    "type": "integer",
+                    "description": "Recorded value that caused the schema violation"
+                  },
+                  "expected": {
+                    "type": "integer",
+                    "description": "Maximum or minimum value allowed"
+                  }
+                },
+                "required": [ "schemaKey", "instance", "igluRepositories", "property", "value", "expected" ],
+                "additionalProperties": false
+              },
+              {
+                "properties": {
+                  "etc": {
+                    "type": "string"
+                  }
+                },
+                "required": [ "etc" ],
+                "additionalProperties": false
+              }
```

Comment:
this adt will depend on the iglu client work

Reply:
Can you elaborate @BenFradet ?

Reply:
it will depend on what information is going to be made available in the next iglu client release

Reply:
Understood!